fdbserver/BulkDumpUtil.actor.cpp

```diff
@@ -254,22 +254,27 @@ ACTOR Future<Void> uploadBulkDumpFileSet(BulkDumpTransportMethod transportMethod
                                          BulkDumpFileSet sourceFileSet,
```

Comment:
We want to change similar things at uploadBulkDumpJobManifestFile() as well. This function uploads the global job manifest file to the remote folder.